docker/Dockerfile

```diff
@@ -58,11 +36,16 @@ FROM ubuntu:$UBUNTU
 COPY --from=builder /snap/core /snap/core
 COPY --from=builder /snap/core18 /snap/core18
 COPY --from=builder /snap/core20 /snap/core20
+COPY --from=builder /snap/core20 /snap/core22
 COPY --from=builder /snap/snapcraft /snap/snapcraft
 COPY --from=builder /snap/bin/snapcraft /snap/bin/snapcraft
 
 # Generate locale and install dependencies.
-RUN apt-get update && apt-get dist-upgrade --yes && apt-get install --yes snapd sudo locales && locale-gen en_US.UTF-8
+RUN apt-get update && apt-get -y dist-upgrade && apt-get -y install \
```

Comment:
Minor: not sure whether this change is really necessary as it just changes the order of arguments and adds lines

Reply:
It is Docker best practices https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#sort-multi-line-arguments

Reply:
Yeah, following best practices is a good thing, but it's not mentioned in the commit message and looks unrelated (could be in another commit and PR)